tests/rule_based_toolchain/legacy_features_as_args/goldens/windows/libraries_to_link.textproto

```diff
@@ -0,0 +1,117 @@
+enabled: false
```

Comment:
I did not compare this to the legacy textproto, I'm not sure how to do that, but I'm sure it will look different if the legacy assumes only MSVC